test/Dialect/FxpMathOps/lower-uniform-real-math-mulew.mlir

```diff
@@ -88,7 +88,7 @@ func @real_mulew_unquantized_result(%arg0 : !type_lhs, %arg1: !type_rhs) -> !typ
 !type_rhs = type tensor<4x!quant.uniform<i8:f32, 3.875e-2>>
 !type_result = type tensor<4x!quant.uniform<i8:f32, 1.065e-4>>
 func @real_mulew_multiplier_gt_1(%arg0 : !type_lhs, %arg1: !type_rhs) -> !type_result {
-  // expected-warning@+1 {{unimplemented: cannot multiply with multipler > 1.0}}
+  // expected-warning@+1 {{unimplemented: cannot multiply with multiplier > 1.0}}
```

Comment:
Unfortunately this test cannot pass if you don't update the producer of the error message in lib/Dialect/FxpMathOps/Transforms/LowerUniformRealMath.cpp

Reply:
I suspect having a single pull-request for #177 and this one may be easier.

Reply:
I suspect having a single pull-request for #177 and this one may be easier, but feel free to pull just these changes if you prefer (or move the change to this test into #177)

Reply:
Thank you for pointing out the dependency. I will move this change in `test/Dialect/FxpMathOps/lower-uniform-real-math-mulew.mlir` into #177.

Reply:
The reason I created three PRs is easy to review since more than 100 changes exist.   
Since it is easy for me to move one change from #175 to #177, let me keep three PRs now.